backends/ebpf/runtime/pcap_util.c

```diff
@@ -41,7 +41,7 @@ pcap_list_t *append_packet(pcap_list_t *pkt_list, pcap_pkt *pkt) {
     pkt_list->pkts = realloc(pkt_list->pkts, pkt_list->len * sizeof(pcap_pkt *));
     if (pkt_list->pkts == NULL) {
         fprintf(stderr, "Fatal: Failed to expand the"
-            "packet list with size %u !\n", pkt_list->len);
+            "packet list with size %lu !\n", pkt_list->len);
```

Comment:
Is the list length a long?
There is no reason for it to be a long. Use either unsigned or size_t.

Reply:
Length is u64, I guess technically llu would be a better format. Pcap files and the amount of packets can be arbitrarily large, so I thought it might be sensible to be liberal in index size.

Reply:
I guarantee you that you will never have more than 2 billion packets in a file.
Certainly not in our testing cases.